<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>время</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="picture_Map"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="set_line_map">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>670</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Растояние</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_length_map">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Длина карты</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="line_map">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>710</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_map">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить карту</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lenght_map">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
